### ISSUE TITLE
New Message Badge / Notifiication New Unread Message

### ISSUE THREAD

**Issue by kopianan** [flyerhq/flutter_firebase_chat_core#125]
<img width="250" alt="image" src="https://github.com/flyerhq/flutter_firebase_chat_core/assets/34189066/1aadd62c-84b9-4b5e-9f8b-a41166933254">

Anyone know how to make this badge ? 
i don't see this feature. 
Is this need cloud function ? i think yes

**Comment by demchenkoalex** [comment]
Yes you need a cloud function. I will be archiving this repo soon, so I recommend you copy the code if you want and then you can do any logic you want, including this with cloud functions.
